src/vue/backend/src/templates/listingsCard.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <checkbox-label
+        :option="option"
+        @checked="$emit('listingSelected', [$event.data, property.listingID])"
+    >
+        <template v-slot:content>
+            <!-- Prop bindings will need to be reevaluated once we have the property shape -->
+            <idx-single-property-card
+                customClass="import-listing__card"
+                :listingPrice="property.listingPrice"
+                :bedrooms="property.bedrooms"
+                :totalBaths="property.totalBaths"
+                :sqFt="property.sqFt"
+                :address="property.address"
+                :cityName="property.cityName"
+                :courtesy="property.courtesy"
+                :image="property.image[1]"
```

Comment:
If we don't have guards to check if `property.image` and `property.image[1]` exist for lines 17 and 18, then we'll get runtime errors when it doesn't